### PR TITLE
Create buildings for all housing company addresses

### PR DESCRIPTION
**Pull request by matti-lamppu**
# Hitas Pull Request

# Description

Removes street address from real estate and adds buildings to the first real estate from all housing company addresses during migration.

## Pull request checklist

Check the boxes for each DoD item that has been completed:

- **Testing**
  - [x] Changes have been tested
  - [x] Automatic tests have been added
- **Database**
  - [ ] Database migrations will work in the DEV & TEST environments
  - [x] initial.json has been updated to work with migrations
  - [ ] Oracle migration has been updated
- **Documentation**
  - [ ] Tooltips have been added in the frontend for all new fields
  - [x] OpenAPI definitions have been updated
  - [ ] Test instructions have been written for the customer in the appropriate ticket in Jira
  - [ ] Terminology page in Confluence has been updated

## Test plan

- [x] Automated tests
- [ ] Run migration without issues
- [ ] Perform some operations on real estates in the frontend

## Tickets

This pull request resolves all or part of the following ticket(s): HT-190

### CODE REVIEW

**Review by ranta — APPROVED** [review]
🦝

File: backend/hitas/migrations/0075_remove_realestate_street_address.py
Comment (ranta) [inline backend/hitas/migrations/0075_remove_realestate_street_address.py:8]: Remember to fix this file migration number before merging this PR

For future reference, if you want separate PRs, but you already have a PR open that the second one is dependent (e.g. both create new migration files) you can have the second PR branch from the first PR's last commit, and then open the second PR to the first PR. That way when the first PR is merged, the second PR's target branch is automatically switched to `master`, and you dont need to remember to fix migration files 😉

Comment (matti-lamppu) [reply backend/hitas/migrations/0075_remove_realestate_street_address.py:8]: I opened separate PRs since either one can be merged first, and if one PR gets more comments than the other, it doesn't then block the merge of the other. Anyway, it's not ideal to work on multiple PRs that depend on each other at the same time, since if there are any conflicting changes to be made, it makes the rebase work harder. This can't be avoided always, but I'd rather have less of it than more.

**Pull request merged**
